frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizations.tsx

```diff
@@ -185,29 +224,30 @@ const PendingOrganizations = ({
               )
             : ""}
         </td>
-        <td data-testid="org-created-at-table-cell">
+        <td
+          data-testid="org-created-at-table-cell"
+          className="word-break-break-word"
+        >
           {new Date(o.createdAt).toLocaleString()}
         </td>
-        <td>{o.externalId}</td>
-        <td className="verify-button-container">
+        <td className="word-break-break-word">{o.externalId}</td>
+        <td className="word-break-normal">
```

Comment:
don't include `word-break: "break-word"` here otherwise the button text breaks into another line and does not look pretty 😅